src/components/SQLEditor/SQLEditor.test.tsx

```diff
@@ -80,7 +80,11 @@ const defaultProps: SQLEditorProps = {
 const setup = async (props: Partial<SQLEditorProps> = {}) => {
   const combinedProps = { ...defaultProps, ...props };
 
-  const renderResult = render(<SQLEditor {...combinedProps} />);
+  const renderResult = render(
+    <SchemaTreeContextProvider>
```

Comment:
Why not add `SchemaTreeContextProvider` inside `render` method? Just to completely emulate the wrappers around all the tests.

Reply:
Yes, great suggestion. Implementing now.